pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerGauge.java

```diff
@@ -41,7 +41,7 @@
   REALTIME_OFFHEAP_MEMORY_USED("bytes", false),
   REALTIME_SEGMENT_NUM_PARTITIONS("realtimeSegmentNumPartitions", false),
   LLC_SIMULTANEOUS_SEGMENT_BUILDS("llcSimultaneousSegmentBuilds", true),
-
+  EXECUTION_THREAD_CPU_TIME_NS("executionThreadCpuTimeNs", false),
```

Comment:
We should pass the unit as the first argument.

Reply:
done